examples/cat-rest-api/src/cats/cat.controller.spec.ts

```
@@ -0,0 +1,85 @@
+import supertest from "supertest";
```

ESLint

[error] Unable to resolve path to module 'supertest'.

(import-x/no-unresolved)